.github/workflows/test.yml

```
@@ -723,16 +723,16 @@ jobs:
 
       - name: Upload test results to BuildPulse for flaky test detection
         # Only run this step for branches where where we have access to secrets.
-        # Only run this step on macOS for now.
         # Run this step even when the tests fail. Skip if the workflow is cancelled.
-        if: startsWith(github.repository, 'prisma/') && runner.os == 'macOS' && !cancelled()
-        env:
-          BUILDPULSE_ACCESS_KEY_ID: ${{ secrets.BUILDPULSE_ACCESS_KEY_ID }}
-          BUILDPULSE_SECRET_ACCESS_KEY: ${{ secrets.BUILDPULSE_SECRET_ACCESS_KEY }}
-        run: |
-          curl -fsSL https://get.buildpulse.io/test-reporter-darwin-amd64 > ./buildpulse-test-reporter
-          chmod +x ./buildpulse-test-reporter
-          ./buildpulse-test-reporter submit packages --account-id 17219288 --repository-id 192925833
+        if: startsWith(github.repository, 'prisma/') && !cancelled()
```

This could/should work maybe:
```suggestion
        if: github.event.pull_request.head.repo.full_name == github.repository && !cancelled()
```